.travis.yml

```diff
@@ -2,32 +2,32 @@
 sudo: false
 language: python
 env:
-  - TOXENV=py27-django18-sqlite
-  - TOXENV=py27-django18-mysql
-  - TOXENV=py27-django18-postgres
+  - TOXENV=py27-django18-sqlite PYTHONPATH=$HOME/virtualenv/python2.7.9/lib/python2.7/site-packages  TOX_TESTENV_PASSENV="PYTHONPATH"
```

Comment:
It'd be good to document with a comment why these envvars need to be set as one would think tox already takes care of the `PYTHONPATH`.